Makefile

```
@@ -32,12 +32,50 @@ else
 endif
 
 test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (11).

(maxbodylength)